app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   include SessionsHelper
   before_action :set_locale
+  helper_method :current_receipt
```

Comment:
Tại sao lại cần khai báo file này

Reply:
session[:cart] = {}
session[:cart] = {room_id: time}  #time{form: , to: }
session[:cart] = {1: x, 2: y} 
id room -> exist DB
ko exist: xoa id room -> session + message bao cho ng dung....